IncrementSlider/tests/IncrementSlider-specs.js

```diff
@@ -100,474 +98,398 @@ describe('IncrementSlider Specs', () => {
 	});
 
 	test('should disable decrement button when value === min', () => {
-		const incrementSlider = mount(
+		render(
 			<IncrementSlider
 				value={0}
 				min={0}
 			/>
 		);
 
-		const expected = true;
-		const actual = incrementSlider.find(`IconButton.${css.decrementButton}`).prop('disabled');
+		const expected = 'disabled';
+		const actual = screen.getAllByRole('button')[0];
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveAttribute(expected);
 	});
 
 	test('should disable increment button when value === max', () => {
-		const incrementSlider = mount(
+		render(
 			<IncrementSlider
 				value={10}
 				max={10}
 			/>
 		);
 
-		const expected = true;
-		const actual = incrementSlider.find(`IconButton.${css.incrementButton}`).prop('disabled');
+		const expected = 'disabled';
+		const actual = screen.getAllByRole('button')[1];
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveAttribute(expected);
 	});
 
 	test('should use custom incrementIcon', () => {
 		const icon = 'plus';
-		const incrementSlider = mount(
+		render(
 			<IncrementSlider incrementIcon={icon} />
 		);
 
-		const expected = icon;
-		const actual = incrementSlider.find(`.${css.incrementButton} Icon`).prop('children');
+		const expected = '+';
+		const actual = screen.getAllByRole('button')[1].children[1].children[0];
```

Comment:
I believe you can remove the .children[1].children[0] from here and the test will do the same thing,